src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -56,6 +56,11 @@ public class Win32PlatformOptions
         /// GPU rendering will not be enabled if this is set to false.
         /// </remarks>
         public bool? AllowEglInitialization { get; set; }
+        
+        public IList<string> EglRendererBlacklist { get; set; } = new List<string>
```

Comment:
IReadOnlyList<T>?

Reply:
it could be useful to add one in runtime?

Reply:
I think at runtime it is not useful. If I need to add a render to the blaklist, I will always have to restart the app to make it active.

Reply:
I think at runtime it is not useful. If I need to add a render to the blaklist, I will always have to restart the app to make it active.